android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -88,6 +89,21 @@ public void screen(String event, ReadableMap properties, ReadableMap options) {
                 .build());
     }
 
+    @ReactMethod
+    public void putAndroidDeviceToken(String token) {
+        if (rudderClient == null) return;
+        RudderContext rudderContext = rudderClient.getRudderContext();
+        if (rudderContext != null && token != null) {
+            rudderContext.putDeviceToken(token);
+        }
+    }
+
+    @ReactMethod
```

Comment:
Should be removed as per the previous comment

Reply:
done